pages/linux/arp-scan.md

```diff
@@ -0,0 +1,19 @@
+# arp-scan
+
+> Send ARP packets to hosts on the local network to scan which ones answer.
+
+- Scan the current local network:
+
+`arp-scan --localnet`
+
+- Scan an IP network with a custom bits mask:
```

Comment:
Shouldn't this be "bitmask"?

Reply:
I think you're right, I'll fix this when I have some free time!

Reply:
Sure! Just @mention me in a comment once you've made the changes.